virttest/qemu_qtree.py

```diff
@@ -594,6 +594,18 @@ def check_drive_format(node, params):
                     self.errors.append(error_msg)
                     err += 1
             disks.pop(qname)
+
+        if params.get('ovmf_path'):
+            ovmf_code_filename = params["ovmf_code_filename"]
+            ovmf_code_path = os.path.join(params['ovmf_path'],
+                                          ovmf_code_filename)
+            path = storage.get_image_filename_filesytem(params,
+                                                        data_dir.get_data_dir())
+            ovmf_vars_path = "%s.fd" % path
+            for (qname, disk) in disks.items():
```

Comment:
Suggest use `iteritems`, as `items` has lower performance when compare with `interitems`.
and `six.iteritems ` to cover both python 2 and python 3.   it is up to you.

Reply:
@yanan-fu I can't user iteritems because I will pop() the item inside the loop, which will change the dictionary, that is not allowed for iteritems.

Reply:
@vivianQizhu  yes, your right, thanks !